### PR TITLE
docs - show latest version on landing page

### LINKED ISSUE #577: Show the latest version on the landing page?
Would it be useful to show the latest version number on the landing page? And maybe the last push date?

### PR DESCRIPTION
**Pull request by stephenplusplus**
Fixes #577

Preview: http://stephenplusplus.github.io/gcloud-node/#/

screen shot 2015-05-19 at 4 30 17 pm

### CODE REVIEW

File: docs/site/home.html [hunk 1]
```diff
@@ -15,8 +15,11 @@ <h1 class="logo"><img src="site/img/logo-full.svg" alt="Google Cloud Platform" /
       </div>
 
       <div class="quote-box--supplementary">
-        <h2>One-line npm install</h2>
```
Comment (stephenplusplus) [inline docs/site/home.html]: @jgeewax look what I found:

screen shot 2015-05-19 at 2 46 44 pm

Somewhere along the way, the font color must have been changed to match the background. Or, it was never visible from the start.

Python is also a victim...

screen shot 2015-05-19 at 2 47 34 pm

... while Ruby doesn't seem to have it. I don't think the line is necessary, what do you think?

File: docs/site/home.html [hunk 2]
```diff
@@ -15,8 +15,11 @@ <h1 class="logo"><img src="site/img/logo-full.svg" alt="Google Cloud Platform" /
...
-        <pre>$ npm install --save gcloud</pre>
+        <pre><code class="subtle--blue">$</code> npm install <code class="subtle--blue">--save</code> <strong>gcloud</strong></pre>
+        <h4 class="latest-release subtle" ng-if="latestRelease">
+          Latest Release <strong class="white">{{latestRelease}}</strong>
```
Comment (ryanseys) [inline docs/site/home.html]: Can we wrap the version in a link to either `npm` page for our package or the specific changelog docs for the release on GitHub?

Comment (stephenplusplus) [inline docs/site/home.html]: I went with linking to the release page on GitHub, but if you think another option is better, it doesn't matter much to me. I personally never go to a package on npmjs.org, and if I do end up there, I quickly find the link to GitHub, as on GH, I get the same content with the benefit of code and an ultra-familiar interface. (Not counting the metadata from npm that I don't find too interesting)

Comment (ryanseys) [inline docs/site/home.html]: I'm cool with this. Gives my release docs more air time :)

Comment (stephenplusplus) [inline docs/site/home.html]: Haha, they are the best release notes on GH, pretty sure.

**Pull request merged**